packages/fhi-designsystem/src/components/fhi-grid/fhi-grid.stories.ts

```diff
@@ -0,0 +1,127 @@
+import type { Meta, StoryObj } from '@storybook/web-components';
+import { html } from 'lit';
+import { FhiGrid } from './fhi-grid.component';
+import { FhiButton } from '../fhi-button/fhi-button.component';
+import { ifDefined } from 'lit/directives/if-defined.js';
+
+new FhiGrid();
+new FhiButton();
+
+const meta: Meta<FhiGrid> = {
+  title: 'Komponenter/Grid',
+  component: 'fhi-grid',
+  parameters: {
+    options: { selectedPanel: 'addon-controls' },
+    docs: {
+      source: {
+        excludeDecorators: true,
+      },
+    },
+  },
+  decorators: [],
+  render: args => html`<fhi-grid gap=${args.gap}></fhi-grid>`,
+  argTypes: {
+    gap: {
+      control: 'text',
+      description:
+        'Bestemmer avstand mellom elementene. Kan være `small`, `medium`, `large` eller en spesifikk verdi som `10px`, `1rem` eller `20`.',
+      defaultValue: { summary: 'medium' },
+    },
+    rows: {
+      controls: 'number',
+      description: 'Bestemmer antall rader.',
+    },
+    columns: {
+      controls: 'number',
+      description: 'Bestemmer antall kolonner',
+      defaultValue: { summary: '12' },
+    },
+  },
+};
+
+type Story = StoryObj<FhiGrid>;
+
+export const Preview: Story = {
+  tags: [],
+  args: {
+    gap: 'medium',
+    rows: 3,
+    columns: 10,
```

Comment:
Siden 10 er utenfor våre anbefalte verdier, tenker jeg vi justerer denne til 12, så vi viser det vi selv mener er "best practice" først.

```suggestion
    columns: 12,
```